.goreleaser.yaml

```diff
@@ -198,4 +199,6 @@ changelog:
   use: github-native
 
 release:
-  draft: true
```

Comment:
[Kuba mentioned](https://github.com/opentffoundation/opentf/pull/173#discussion_r1305684450) it would be good to create the release as draft. @cube2222 Does it still hold true?

Reply:
Creating as draft generate the artifacts but doesn't list them in the release page

Reply:
I'd love to have it as a draft first, but if it doesn't work, then it doesn't work 🤷‍♂️